UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs

```diff
@@ -133,6 +133,7 @@ public float GetMoles(Gas gas)
 		}
 
 		public void ChangeVolumeValue(float value){
+			var TemperatureCash = Temperature;
```

Comment:
that was just leftovers from another equation That I was planning To implement, It would take all of the energy inside of the volume and then put it back into the Resized volume, getting you the correct temperature and pressure